### ISSUE TITLE
Think of a cool logo?

### ISSUE THREAD

**Issue by eduardo-rodrigues** [scikit-hep/histoprint#47]
There are very nice example in Scikit-HEP from where to take inspiration :-).

**Comment by ast0815** [comment]
Hmm, maybe combine a figlet version of "histoprint" with an actual histogram?

https://manytools.org/hacker-tools/ascii-banner/

**Comment by eduardo-rodrigues** [comment]
Yes, that would be nice given the nature of the package, meaning what it is meant to do. While at it, remember the grey used e.g. in uproot.

**Comment by eduardo-rodrigues** [comment]
Hello @ast0815, browsing open issues in the repo ... Is this something you could deal with and finalise soon?

**Comment by ast0815** [comment]
I do not have spare time for this at the moment. I can come back to this at the end of June.

**Comment by eduardo-rodrigues** [comment]
Fair enough. Just ensuring this library does not get too dormant and unmaintained, as per our chat at the org admins meeting last week. This one task seemed like a fun one "to relax". Many thanks for the quick reply.

Good luck with the work.

**Comment by ast0815** [comment]
I added a simple logo to `logo`.

**Comment by eduardo-rodrigues** [comment]
It looks good 👍.

I think you should do a couple of things so that the logo is actually visible:
- Update the README so that it is displayed, as we do for the other packages.
- Update the relevant YAML for your package on the org site so that is gets displayed for example at https://scikit-hep.org/packages/.

**Comment by ast0815** [comment]
I am not actually allowed to create a branch (for a PR) on [scikit-hep/scikit-hep.github.io](https://github.com/scikit-hep/scikit-hep.github.io). How should I do this?

**Comment by eduardo-rodrigues** [comment]
That's really strange, especially that I see you as a member of the org at https://github.com/orgs/scikit-hep/people (though I see you're set as private, hence not visible as an org member, which is a shame).

Sure you cannot just clone the repo and work as usual from your clone creating a branch and pushing to origin?

**Comment by ast0815** [comment]
I never consciously set that to "private". Changed now. ;)

I was able to clone it without problems, but when I try to push 
something, I get this:

~/misc/scikit-hep.github.io feature/histoprint_logo
❯ git remote -v
origin	***@***.***:scikit-hep/scikit-hep.github.io.git (fetch)
origin	***@***.***:scikit-hep/scikit-hep.github.io.git (push)

~/misc/scikit-hep.github.io feature/histoprint_logo
❯ git push --set-upstream origin feature/histoprint_logo
ERROR: Permission to scikit-hep/scikit-hep.github.io.git denied to ast0815.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

I also cannot see the "Settings" page of that repo on GitHub, so I 
assume I just don't have the rights.

Lukas

**Comment by eduardo-rodrigues** [comment]
I added you explicitly with write karma. But I'm still puzzled. I would simply do "git push origin feature/histoprint_logo" if "feature/histoprint_logo" is your local branch.

**Comment by ast0815** [comment]
Now it works. I created a PR, since I was not sure whether I should directly edit the live website.

**Comment by eduardo-rodrigues** [comment]
We almost always work via PRs except for trivialities such as a typo fix and alike. Many thanks, will look as soon as done with my long meeting afternoon ...

**Comment by eduardo-rodrigues** [comment]
Close via https://github.com/scikit-hep/histoprint/commit/8d01a2629b8d3834c3bd9b4e6641141a36b88be3, https://github.com/scikit-hep/histoprint/commit/e8470b930db41d1461100555e1c0fe7b75a30d59 and https://github.com/scikit-hep/scikit-hep.github.io/pull/466.
